build.sbt

```diff
@@ -69,24 +104,47 @@ lazy val scanamo = (project in file("scanamo"))
 
       "org.scalatest" %% "scalatest" % "3.0.1" % Test,
       "org.scalacheck" %% "scalacheck" % "1.13.4" % Test
-    ),
+    )
+  )
+  .dependsOn(formats)
 
-    startDynamoDBLocal := startDynamoDBLocal.dependsOn(compile in Test).value,
-    test in Test := (test in Test).dependsOn(startDynamoDBLocal).value,
-    testOptions in Test += dynamoDBLocalTestCleanup.value,
+lazy val alpakka = (project in file("alpakka"))
+  .settings(
+    commonSettings,
+    publishingSettings,
+    dynamoTestSettings,
 
-    doctestMarkdownEnabled := true,
-    doctestDecodeHtmlEntities := true,
-    doctestTestFramework := DoctestTestFramework.ScalaTest,
+    name := "scanamo-alpakka",
+  )
+  .settings(
+
+    libraryDependencies ++= Seq(
+      "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.11.190",
+      "org.typelevel" %% "cats-free" % "1.0.0-MF",
+      "com.lightbend.akka" %% "akka-stream-alpakka-dynamodb" % "0.11",
 
-    parallelExecution in Test := false
+      "org.scalatest" %% "scalatest" % "3.0.1" % Test,
+      "org.scalacheck" %% "scalacheck" % "1.13.4" % Test
+    ),
+
+    fork in Test := true,
+    envVars in Test := Map(
+      "AWS_ACCESS_KEY_ID" -> "dummy",
+      "AWS_SECRET_KEY" -> "credentials"
+    ),
+    dynamoDBLocalDownloadDir := file(".alpakka-dynamodb-local"),
+    dynamoDBLocalPort := 8052,
```

Comment:
Wish I don't need this but when it can conflicts with `scanamo` module tests when both modules tests are run at the same time.
There's probably a way to run the tests sequentially between modules though